rasa/core/processor.py

```diff
@@ -290,11 +290,22 @@ def _log_slots(tracker):
         if slot_values.strip():
             logger.debug(f"Current slot values: \n{slot_values}")
 
+    def _get_keywordlist():
+        keywordlist = ["restart", "back", "stop", "start", "action"]
```

Comment:
Where are these keywords coming from? Ideally they would be imported from one of our `constants.py` files. But as far as I know only `/restart` and `/back` are defined

Reply:
Okay, I will fix it and import data from constants.py